adafruit_requests.py

```diff
@@ -366,6 +360,60 @@ def __init__(
         self._session_id = session_id
         self._last_response = None
 
+    def _build_boundary_data(self, files: dict):
```

Comment:
this needs tests

Reply:
For this, I would probably build out a couple payloads (5 or 6 all different combinations) and send them with CPython requests, and make sure what is sent matches exactly